src/scrabble.js

```diff
@@ -1,5 +1,48 @@
-function scrabble() {
-  // write code here
+const scrabbleValue = {
+  A: 1,
+  E: 1,
+  I: 1,
+  O: 1,
+  U: 1,
+  L: 1,
+  N: 1,
+  R: 1,
+  S: 1,
+  T: 1,
+  D: 2,
+  G: 2,
+  B: 3,
+  C: 3,
+  M: 3,
+  P: 3,
+  F: 4,
+  H: 4,
+  V: 4,
+  W: 4,
+  Y: 4,
+  K: 5,
+  J: 8,
+  X: 8,
+  Q: 10,
+  Z: 10
+}
+
+const scrabble = (makeWord) => {
+  if (
+    makeWord === `    ` ||
+    makeWord === ` \t\n` ||
+    makeWord === null ||
+    typeof makeWord !== `string`
+  ) {
+    return 0
+  }
+  const printWord = makeWord.toUpperCase()
+  let sum = 0
+
+  for (let i = 0; i < makeWord.length; i++) {
+    sum += scrabbleValue[printWord[i]]
+  }
+  return sum
```

Comment:
Definitely not going to say anything to improve since i'm new to this but one thing i learnt myself is that you can use " if(!makeWord) return 0 " and this would of done your if statement for you i think?

Reply:
i will give that a shot!